include/boost/unordered/concurrent_flat_map.hpp

```diff
@@ -471,19 +473,21 @@ namespace boost {
       }
 
       template <class InputIterator, class F>
-      void insert_or_visit(InputIterator first, InputIterator last, F f)
+      size_type insert_or_visit(InputIterator first, InputIterator last, F f)
       {
         BOOST_UNORDERED_STATIC_ASSERT_INVOCABLE(F)
-        for (; first != last; ++first) {
+        size_type count = 0;
```

Comment:
GCC 7 complains that this variable shadows `count` member function. Happens passim.

https://github.com/boostorg/unordered/actions/runs/11045334436/job/30682893588?pr=287